.circleci/config.yml

```diff
@@ -118,7 +118,14 @@ commands:
   codegen_test_configurations_generate_and_test:
     steps:
       - run:
-          command: git clone -b "$CIRCLE_BRANCH" "$CIRCLE_REPOSITORY_URL"
+          command: |
+            if [[ ! -z "${CIRCLE_PR_REPONAME}" ]]; then
+              git clone "https://github.com/$CIRCLE_PR_USERNAME/$CIRCLE_PR_REPONAME.git" apollo-ios
```

Comment:
Forked repo PRs are not as straightforward as local PRs and we have to build the parameters for clone and checkout from environment variables.